surefire-providers/surefire-junit-platform/src/test/java/org/apache/maven/surefire/junitplatform/RunListenerAdapterTest.java

```diff
@@ -630,9 +639,9 @@ private static TestIdentifier newSourcelessChildIdentifierWithParent(
         when(child.getSource()).thenReturn(Optional.empty());
         when(child.getParent()).thenReturn(Optional.of(parent));
         TestIdentifier childId = TestIdentifier.from(child);
-
-        testPlan.add(childId);
-        testPlan.add(parentId);
+        // TODO https://github.com/junit-team/junit5/issues/1732
```

Comment:
not sure to understand as it is a closed issue.
what's left to do?

Reply:
i'm not sure if my proposal to change from .add -> .addInternal is correct

Reply:
In this case maybe just add a comment in the PR because leaving this in the code feels like there is something left to do :) 
Anyway looks good